src/Annotation/ApiResource.php

```diff
@@ -274,128 +312,127 @@ final class ApiResource
      *
      * @var array
      */
+    private $openapiContext;
```

Comment:
This one is https://api-platform.com/docs/core/swagger/#using-the-openapi-and-swagger-contexts